src/ImageOptimizationLambda.ts

```diff
@@ -43,15 +43,16 @@ export class ImageOptimizationLambda extends Function {
 
     const code = isPlaceholder
       ? Code.fromInline(
-        "module.exports.handler = async () => { return { statusCode: 200, body: 'SST placeholder site' } }"
-      )
+          "module.exports.handler = async () => { return { statusCode: 200, body: 'cdk-nextjs placeholder site' } }"
+        )
       : Code.fromAsset(props.nextBuild.nextImageFnDir);
 
     super(scope, id, {
       code,
       handler: 'index.handler',
       runtime: LAMBDA_RUNTIME,
       architecture: Architecture.ARM_64,
+      description: 'Next.js Image Optimization Function',
```

Comment:
https://dev.to/dashbird/4-tips-for-aws-lambda-optimization-for-production-3if1

Can you change the default memory size to 1536mb please? That is faster and cheaper than 1024

Reply:
Does this apply to other functions as well? e.g. https://github.com/kevin-mitchell/cdk-nextjs/blob/e60c3bfb5dc00d1871cea18de6ad3b91b29c1c79/src/NextjsS3EnvRewriter.ts#L76 or https://github.com/kevin-mitchell/cdk-nextjs/blob/e60c3bfb5dc00d1871cea18de6ad3b91b29c1c79/src/NextjsLambda.ts#L90 ?

I would think it would be appropriate to make a constant for this value and share it across Lambda functions. Based on the linked article I'm assuming it would be best to have them all at 1536.

Reply:
Yes that'll be a good idea.